lib/eyeshade/overall_balance.rb

```diff
@@ -0,0 +1,27 @@
+# Fees applied to channel balances
+module Eyeshade
+  class OverallBalance < BaseBalance
+    def initialize(rates, default_currency, accounts)
+      super(rates, default_currency)
+
+      @amount_probi = 0
```

Comment:
It might make sense to have this property be able to be set in the `BaseBalance` class and then the `amount_bat` or `amount_probi` can be calculated based upon what is already defined in the class.

Although I suppose the next question is that what if there is both an `amount_probi` set and we're trying to set `amount_bat`. 

Maybe it's best to keep it like this for now and not assume anything?

Reply:
Fixed. 

@nvonpentz I agree, I think we should only track either `amount_probi` or `amount_bat` rather than tracking the same data twice.